R/mod-vax.R

```diff
@@ -30,6 +33,16 @@ vax_covid <- function(dat, at) {
     }
   }
 
+  idsVax1.gt65 <- which(active == 1 & vax == 1 & vax1Time == at & age >= 65)
+  nidsVax1.gt65 <- length(idsVax1.gt65)
+
+  idsVax1.15to65 <- which(active == 1 & vax == 1 & vax1Time == at & age < 65 &
+                        age >= 15)
+  nidsVax1.15to65 <- length(idsVax1.15to65)
+
+  idsVax1.lt15 <- which(active == 1 & vax == 1 & vax1Time == at & age < 15)
```

Comment:
@kkrupins this is fine for now, but just a pointer for the future that you simplify this with:

```r
sum(active == 1 & vax == 1 & vax1Time == at & age < 15)
```